.github/workflows/continuous-integration.yml

```diff
@@ -119,6 +119,33 @@ jobs:
       - name: Generate coverage report by app
         run: node script/app-coverage-report.js > test-results/coverage_report.txt
 
+      - name: Configure AWS credentials (1)
+        uses: aws-actions/configure-aws-credentials@v1
```

Comment:
Should probably add the `if: ${{ github.ref == 'refs/heads/master' }}` to each of these steps too.